src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/ViewModels/Teams/TeamNameViewModel.cs

```diff
@@ -58,6 +66,19 @@ public static TeamNameViewModel Map(Team team)
             };
         }
 
+        /// <summary>
+        /// Returns team's group name
+        /// </summary>
+        /// <param name="team">Mapped team</param>
+        /// <param name="divisions">Divisions list in the current tournament</param>
+        /// <returns>String which contains group name</returns>
+        public static string GetGroupName(TeamNameViewModel team, ICollection<Division> divisions)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Consider using more general type 'System.Collections.Generic.IEnumerable<VolleyManagement.Domain.TournamentsAggregate.Division>' instead of 'System.Collections.Generic.ICollection<VolleyManagement.Domain.TournamentsAggregate.Division>'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3242)